zerver/openapi/zulip.yaml

```diff
@@ -1085,7 +1085,9 @@ paths:
                                 }
                             - type: object
                               description: |
-                                No longer sent by modern Zulip servers.
+                                Event sent to guest users when they lose access to a user.
+
+                                **Changes**: Prior to Zulip 8.0 (feature level 227), this event was deprecated.
```

Comment:
Tbh, I am not sure how to correctly update the docs here.

Reply:
This seems OK for this PR; though we should probably discuss in "api documentation" polishing these docs now that all these corner cases are implemented. The first step might be adding a /help/ article that we can link to about the concept of "losing access to a user" text, though; @alya FYI.